recipes/yasma/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# install using pip from the whl file provided by PyPi
+
+pip install https://files.pythonhosted.org/packages/75/45/54bb2d8d4138964a94bef6e9afe48b0be4705ba66ac442ae7d8a8dc4ffef/click_option_group-0.5.9-py3-none-any.whl
```

Comment:
you can add https://anaconda.org/conda-forge/click-option-group as run-dependencies